vtex/admin-ui
# NAD-865 - Tag component review

#### What is the purpose of this pull request?

To update the current Tag component following the component review guidelines.

[Design specs are available on this Figma file.](https://www.figma.com/file/sepBhrHViUHyrTnktkuHaW/Admin-UI-%E2%80%93-Components?node-id=656%3A286742)

#### How should this be manually tested?


[Visit this storybook story ](https://tag-review.surge.sh/?path=/story/admin-ui-review-tag--variants)and its [documentation section](https://admin-ui-docs-git-feat-tag-review-admin-team.vercel.app/guidelines/components/tag). Make sure it matches the Figma specs.

### Breaking changes

- The `icon` and `handleDelete` properties were removed.
- The `size` property now accepts `normal` and `large`, instead of `small` and `regular`.

You can use our codemod cli to update the tag's properties for you. [See its README for more information on how to execute the codemod](https://github.com/vtex/admin-ui/tree/main/packages/admin-ui-codemod#admin-ui-codemod).

#### Before
```tsx
import { Tag } from '@vtex/admin-ui'

function Example() {
  return (
    <Tag
      icon={<IconHeart />}
      label="Here goes the label!"
      size="small"
      handleDelete={() => window.alert('Tag deleted')}
    />
  )
}
```

#### After
```tsx
import { Tag } from '@vtex/admin-ui'

function Example() {
  return <Tag label="Short text" size="normal" />
}
```



#### Types of changes
